proto/sync/sync.proto

```diff
@@ -2,28 +2,38 @@ syntax = "proto3";
 
 package sync;
 
+import "google/protobuf/empty.proto";
+import "rpcdb/rpcdb.proto";
+
 option go_package = "github.com/ava-labs/avalanchego/proto/pb/sync";
 
 // Request represents a request for information during syncing.
 message Request {
   oneof message {
-    RangeProofRequest range_proof_request = 1;
-    ChangeProofRequest change_proof_request = 2;
+    GetRangeProofRequest range_proof_request = 1;
+    GetChangeProofRequest change_proof_request = 2;
   }
 }
 
-// A RangeProofRequest requests the key-value pairs in a given key range
-// at a specific revision.
-message RangeProofRequest {
+service SyncableDB {
+  rpc GetMerkleRoot(google.protobuf.Empty) returns (GetMerkleRootResponse);
+
+  rpc GetChangeProof(GetChangeProofRequest) returns (GetChangeProofResponse);
+  rpc VerifyChangeProof(VerifyChangeProofRequest) returns (VerifyChangeProofResponse);
+  rpc CommitChangeProof(CommitChangeProofRequest) returns (CommitChangeProofResponse);
+
+  rpc GetProof(GetProofRequest) returns (GetProofResponse);
+
```

Comment:
I'm assuming CommitProof is going to need to verify the proof before actually committing it.  Do we need/want the separate VerifyChangeProof call as well?

Reply:
>I'm assuming CommitProof is going to need to verify the proof before actually committing it.

Do you mean `CommitChangeProof`? We don't ever commit "regular" (i.e. not range or change) proofs.

> Do we need/want the separate VerifyChangeProof call as well?

It's used by the sync manager so yeah we need this